### ISSUE TITLE
[Drupal] Update footer link for VA Scorecard to Performance Dashboard

### ISSUE THREAD

**Issue by kennedycollins** [department-of-veterans-affairs/va.gov-team#6933]
### Background
*Provide a background of your tool/functionality.*
The ThoughtWorks team was asked to rebuild and refresh the VA.gov scorecard, which has been orphaned for ~18 months. As part of that work, we came to the understanding that "scorecard" wasn't the best name for what we were building, and so we will instead be calling it the VA.gov "Performance Dashboard". Ultimately, we intend to move this to a subdomain, but as an intermediate step to make it clear that this is a new thing, distinct from the old "scorecard", we'll be creating /performance-dashboard and displaying the new site there.
**We specifically need help getting links to the current scorecard in the VA.gov footer (and wherever else is appropriate) changed to point to the new url (www.va.gov/performance-dashboard) with the new name (Performance Dashboard)**

* **Is the request related to a VA.gov tool? Provide a brief description of the tool or functionality:**  
   * The Performance dashboard is a tool used to share information about the performance of various va.gov tools, with the intention of using that information to influence others outside of DEPO/the Office of the CTO.  
* **Provide links to any available collateral: product briefs/outlines, wireframes, prototypes, user research synthesis:** 
   * < links to collateral >

### Request Type
*Select all applicable and provide details behind your request* 

- [ ] **Content support for authenticated app/tool.** Note: Support is provided only for VSA apps teams that do not have its own dedicated content writer. In addition to the generic information above, please share:
   * Word document with draft if available, or link to the URL where the draft content appears:
      * < link > 
   * Provide design context: (Example: Is this content in a user flow, an in-line error message, tool tip, field label?)
      * < design context >
   * Scope: How many screens? (Or messages, etc.) 
      * < scope > 
      
      
- [ ] **Content support for a static public website page**
   * Is this to update existing static content? 
      * < y/n >
   * Does this need a new static tool landing page? (The content team may propose a different approach in consultation with IA.)
      * < y/n >
   * Provide a list of URL/s that need to be reviewed or updated. 
      * < URL list >
   * If it's a small copy section, list what the new info should be and what it is replacing.
      * < list of new info >

- [x] **New URL or Redirect Requests**
   * Have you vetted this request by the VSP IA team?
      * Yes
   * Link to VSP IA Redirect Request ticket
      * https://github.com/department-of-veterans-affairs/va.gov-team/issues/6928      

   
- [ ] **Implementation of REACT sign in widgets**
   * Full spec including different states and behaviors
      * < link to spec >
     
- [ ] **Other**
   * Please describe in detail
      

### Desired/target implementation date
*What is the ETA for the launch or date when request needs to be completed (for instance, for a user research session or eBenefits migration)? Our team's capacity will determine which sprint the work falls into.*

* We're targeting live on or around 3/25; the redirect will happen probably 1-5 business days after that. This could happen any time after launch but there's no real rush as long as the redirect is in place.

### Requestor info
*Indicate what team and who the primary contacts are on your team* 

- Team name: *[ThoughtWorks](@department-of-veterans-affairs/thoughtworks)]
- DEPO product owner: *@clantosswett & @KevinMHoffmanUSDS*
- Product manager: *@rumyeahs with support from @kennedycollins*
- Designer: *@toddstanich* 
- FE engineer: *@ffafara-tw with support from @karlbrown-va*
- Product/team Slack channel: *#va-scorecard-tech (in DSVA slack)*

--- 
#### Next steps
- [ ] follow-up with requestor to address any lingering questions/concerns
- [ ] Create epic/tickets as needed for the Public Websites team to implement

#### Definition of Done
* Request ticket is closed and implementation tickets created as needed

**Comment by lunascoop** [comment]
For Sprint 18:

@RLHecht We need to update content/ text and related link in VA footer: Change "VA Scorecard" to "VA performance dashboard" (see screenshot below). Link will change to www.va.gov/performance-dashboard once the page is live and redirect is implemented.

<img width="1002" alt="Screen Shot 2020-03-20 at 11 11 36 AM" src="https://user-images.githubusercontent.com/42546515/77177482-f4566800-6a9b-11ea-9863-5ecdddb00291.png">

**Comment by RLHecht** [comment]
@lunascoop Just checked and we can't access the footer from Drupal. Is this something @ncksllvn or @kelsonic can work on?

**Comment by ncksllvn** [comment]
@RLHecht Yeah sorry for the confusion, but the footer links are in the front-end code. Kelson or I will have to do it.

**Comment by kelsonic** [comment]
@ncksllvn I'm happy to do this! 🙂

**Comment by lunascoop** [comment]
@ncksllvn Good to know. I assumed that this was something @RLHecht could do. @kelsonic perhaps we should do this when the page is live? (which it is not currently).

**Comment by kelsonic** [comment]
@lunascoop Definitely! I'll get a PR sent up for it now so all we have to do is merge it 👍

**Comment by karlbrown-va** [comment]
Hi, yeah you may want to leave this in a branch until the new URL is live. We are awaiting another PR for this which is here: https://github.com/department-of-veterans-affairs/devops/pull/6558

We also need to actually push the dashboard/scorecard to production, which should happen within a day or two. Will ping you here when we're ready for the new footers.

**Comment by kelsonic** [comment]
[PR is up](https://github.com/department-of-veterans-affairs/vets-website/pull/12012)! We won't merge it until we get the green light from both of you @lunascoop @karlbrown-va 🙂

**Comment by karlbrown-va** [comment]
Note: adding this for reference:
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3419

I note that the scorecard is currently blocked by crawlers. We should decide if this same thing is desired for the performance-dashboard.
https://github.com/department-of-veterans-affairs/vets-website/blob/1b2f8ba5234cbff40d840cfc5df947ccd0c74521/src/site/assets/robots.txt#L96

**Comment by MickinSahni** [comment]
2020-03-26 PW Standup
- Related to #6928 
- PR open for this ticket

**Comment by kelsonic** [comment]
@ncksllvn @RLHecht 

> Note: adding this for reference:
> [department-of-veterans-affairs/vets.gov-team#3419](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3419)
> 
> I note that the scorecard is currently blocked by crawlers. We should decide if this same thing is desired for the performance-dashboard.
> https://github.com/department-of-veterans-affairs/vets-website/blob/1b2f8ba5234cbff40d840cfc5df947ccd0c74521/src/site/assets/robots.txt#L96

@RLHecht @ncksllvn What are your thoughts on blocking it from crawlers? Happy to make the update if desired! 🙂

**Comment by ncksllvn** [comment]
> I note that the scorecard is currently blocked by crawlers. We should decide if this same thing is desired for the performance-dashboard.

I believe `/scorecard` is only blocked by crawlers because the product was rolled over from Vets.gov and just wasn't revisited until now.

But yeah, like @kelsonic said, if your team wants to do some testing in the prod environment before your product is ready to be user-facing, then adding a record into `robots.txt` is an easy way to reduce traffic.

**Comment by kelsonic** [comment]
After asking the `#va-performance-dashboard team what their thoughts are, I've added `Disallow: /performance-dashboard` to robots.txt in the current PR: https://github.com/department-of-veterans-affairs/vets-website/pull/12012/files#diff-f8924c60278e9103e7a46a872bd0e991R97

**Comment by MickinSahni** [comment]
2020-03-30 PW Standup
- Pushing tentative date for when we'll implement redirects/footer changes to 4/2. Waiting for confirmation that page is live.

**Comment by lunascoop** [comment]
@kelsonic Page is live. Footer can be updated and redirected if ready.

**Comment by kelsonic** [comment]
Any update on this?

**Comment by karlbrown-va** [comment]
We have two final issues that were requested before go live, tackling them this week, then I think we can close this out. Will let you know...

**Comment by karlbrown-va** [comment]
@kelsonic we are now live at va.gov/performance-dashboard, this can be merged now.

**Comment by kelsonic** [comment]
PR merged! 🎉 https://github.com/department-of-veterans-affairs/vets-website/pull/12081

**Comment by lunascoop** [comment]
Validated and closing ticket.
